# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/moz.build
# Commit: 08acd8256071
# Full Hash: 08acd8256071ad86345dec3fa66767635b07a4b1
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-09 04:16:57
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 4: Implement the third-party cookie blocking excpetion in CookieService. r=cookie-reviewers,baku,valentin
#   
#   The patch integrates the third-party cookie blocking excpetion in
#   CookieService.
#   
# ==============================================================================

diff -r 3ae9463ee988 -r 08acd8256071 netwerk/cookie/moz.build
--- a/netwerk/cookie/moz.build	Wed Jan 08 19:25:58 2025 +0000
+++ b/netwerk/cookie/moz.build	Wed Jan 08 19:25:59 2025 +0000
@@ -34,6 +34,7 @@
     "CookieServiceChild.h",
     "CookieServiceParent.h",
     "CookieStorage.h",
+    "ThirdPartyCookieBlockingExceptions.h",
 ]
 UNIFIED_SOURCES += [
     "Cookie.cpp",
@@ -48,6 +49,7 @@
     "CookieServiceChild.cpp",
     "CookieServiceParent.cpp",
     "CookieStorage.cpp",
+    "ThirdPartyCookieBlockingExceptions.cpp",
 ]
 XPCSHELL_TESTS_MANIFESTS += [
     "test/unit/xpcshell.toml",